inputFiles/constitutiveDrivers/testPVT_docExample.xml

```diff
@@ -53,24 +53,4 @@
   </Events>
 
   <!-- SPHINX_PVTDRIVER_EVENTS_END -->
-  <!-- Mesh is not used, but GEOSX throws an error without one.  Will resolve soon-->
-  <Mesh>
```

Comment:
were you able to resolver the need for the mesh? I don't see it in this PR but if that's the case, it's great!